recipes/libigl/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, CMake, AutoToolsBuildEnvironment
+
+
+class LibiglConan(ConanFile):
+    name = "libigl"
+    description = ("A Massively Spiffy Yet Delicately Unobtrusive Compression Library "
+                  "(Also Free, Not to Mention Unencumbered by Patents)")
+    topics = ("conan", "libigl", "geometry", "matrices", "algorithms")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libigl.github.io/"
+    license = "MPL2"
```

Comment:
```suggestion
    license = "MPL-2.0"
```

https://spdx.org/licenses/MPL-2.0.html